README.md

```diff
@@ -1 +1,77 @@
-# nvim-package-specification
\ No newline at end of file
+# Plugin metadata specification
+
+The neovim package specification supports a single, top-level package metadata file. This file can be *either* 'plugin.lua' or 'plugin.json'. The format is loosely based on the [Rockspec Format](https://github.com/luarocks/luarocks/wiki/Rockspec-format) and can contain the following fields:
```

Comment:
As I mentioned on #1, what do you think the nomenclature `pluginspec`? We should think of some specific name for this, otherwise we will end up referring to this as at least one of:

 - plugin.json
 - plugin.lua
 - Plugin specification

Other names: `nvimspec`, `neovimspec`, `plugspec`.

Reply:
`nvimspec` and `neovimspec` might sound too much like `vimspec`, a testing framework.

`neospec`? On github there are only two dead projects with zero stars with that name.

Reply:
I personally think `spec` is my favorite right now, but I would be fine with `plugspec` or `pluginspec` too.

It could also be an acronym, eg. `nvs` for `neovimspec`.

Reply:
I lean towards `package specification`, I don't see why we shouldn't be verbose, this is something people will rarely be writing and clarity IMO is the most important thing here.

Reply:
It's not a question of verbosity and more about defining terminology instead of overloading existing terms. If you think about it, "package specification" is actually quite generic and ambiguous; it's just a short description. If someone else creates another "package specification" or "plugin specification", how do we discern between the two?

We also need to consider how using such generic terms may help (or impede) adoption. Will "package specification" ever show results for this project on Google? How do we generally expect plugin authors (new and experienced) to discover this?

Or from another angle, If we were to add some basic vim syntax support, what is the name of the filetype?